packages/expo-yarn-workspaces/common/cross-spawn-sync.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const child_process = require("child_process");
+const os = require("os").platform();
+
+let selectiveCmdify = cmd => cmd;
```

Comment:
inline this logic in `spawnSync`, easier to read with less indirection

Reply:
done